### PR TITLE
fix: engine warning on console for ui

### PR DESCRIPTION
**Pull request by juanpicado**
**Type:** bug

The following has been addressed in the PR:

*  There is a related issue?
*  Unit or Functional tests are included in the PR

**Description:**

```
"engines": {
    "node": ">= 10.13.0",
    "node": ">= 8",
    "npm": ">=5"
  },
```
We still support `v8` and this is a dependency of verdaccio.

Resolves https://github.com/verdaccio/ui/issues/402

### CODE REVIEW

**Comment by DanielRuf** [comment]
@juanpicado doesn't `url-loader` 3 require Node >= 10? So this will still throw a warning or error.

**Comment by juanpicado** [comment]
Yes, it does, but I think nobody is developing on Node 8` anyway. The change https://github.com/webpack-contrib/url-loader/pull/198 does not affect the library behavior.

This is just a dependency and we ship a webpack bundle.  I updated the README.

**Comment by DanielRuf** [comment]
> This is just a dependency and we ship a webpack bundle. I updated the README.

You are right. I saw the other comment about the `devDependency` so this is not relevant for the prod bundles as you wrote =)

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 ## Contributing
 
-We use `>=yarn@1.13.0`, keep on mind we use lock file.
+We use `>=yarn@1.13.0`, keep on mind we use lock file and use at least Node `v10.13.0` to be able to build the project.
```
Comment (DanielRuf) [inline README.md]:
```suggestion
We use `>=yarn@1.13.0`, keep in mind that we use lockfiles and use at least Node `v10.13.0` to be able to build the project.
```

**Pull request merged**